crates/sncast/src/starknet_commands/script.rs

```diff
@@ -458,8 +538,10 @@ pub fn run(
         runtime,
         run_resources: RunResources::default(),
         config,
+        script_ui,
     };
 
+    cairo_hint_processor.script_ui.print("\n");
```

Comment:
I get the idea, but should we do this? will this be consistent with the rest of our codebase? maybe this can be achieved some other way?

Reply:
I don't understand what is the purpose of this.

Reply:
this is to visually separate scarb output and script output

Reply:
reworked